src/DssSpell.t.sol

```diff
@@ -929,7 +893,7 @@ contract DssSpellTest is DssSpellTestBase {
 
     function testSparkSpellIsExecuted() public { // make private to disable
         address SUBPROXY_SPARK = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-        address SPARK_SPELL    = 0x2f2c514137173bc98B3699A0d291f7593637c596;
+        address SPARK_SPELL    = address(0xa3836fEF1D314d4c081C2707a7664c3375F29b61);
```

Comment:
`address()` casting is unnecessary here

Reply:
That is intentional (like in the [config](https://github.com/makerdao/spells-mainnet/blob/c657359a4654427c9b23c803da4d1272532eee40/src/test/config.sol#L95)) to simplify future cleanup. Speaking of previous patterns, you can also find it in 3 previous spells 🙂